docs/getting-started-guides/minikube.md

```diff
@@ -0,0 +1,103 @@
+---
+---
+
+* TOC
+{:toc}
+
+Minikube starts a single node kubernetes cluster locally for purposes of development and testing.
+
```

Comment:
Quick outline of capabilities + incapabilities would be helpful.

Supports:
* kube-dns for service location
* NodePort service Type
* Ingress?

Does not support:
* LoadBalancer service Type